### PR TITLE
Add 1.6 release timeline

### PR DESCRIPTION
**Pull request by annajung**
Signed-off-by: Anna Jung (VMware) <antheaj@vmware.com>

Based on the 1.5 release date, March 10th, this is a proposed timeline for release 1.6.

Following the [17 week timeline](https://github.com/kubeflow/community/blob/master/releases/handbook.md#timeline), this PR proposes the following as the timeline for the 1.6 release:

- **Wednesday, Mar 16th, 2022**: Week 1 - Release Cycle Begins
- **Wednesday, Jun 1st, 2022**: Week 12 - Feature Freeze
- **Wednesday, Jun 15th, 2022**: Week 14 - Manifests Testing Starts
- **Wednesday, Jun 22nd, 2022**: Week 15 - Manifests Testing Ends
- ~**Wednesday, Jun 22nd, 2022**: Week 15 - Docs Update Ends~
- **Wednesday, Jun 22nd, 2022**: Week 15 - Distribution Testing Starts
- **Wednesday, Jul 13th, 2022**: Week 18 - Distribution Testing Ends
- ****Wednesday, Jul 13th, 2022**: Week 18 - Docs Update Ends**
- **Wednesday, Jul 20th, 2022**: Week 19 - Kubeflow v1.6 Released

As discussed during the April 11th release team meeting, due to the upcoming [KubeCon EU (May 16th - 20th)](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/), the proposal adds 2 weeks to the feature freeze deadline, pushing the release to 19 weeks instead of 17 weeks.

Please review and provide feedback by **April 25th, 2022 11:59 pm US Pacific Time**.

cc @kubeflow/release-team @kimwnasptd @kubeflow/wg-pipeline-leads @kubeflow/wg-manifests-leads @kubeflow/wg-notebooks-leads @kubeflow/wg-training-leads @kubeflow/wg-automl-leads @pvaneck @yuzisun @zijianjoy @jbottum

### CODE REVIEW

**Comment by ca-scribner** [comment]
(from Canonical) lgtm!  Thanks @annajung

**Comment by kimwnasptd** [comment]
Thanks for the PR @annajung!

lgtm from Notebooks and Manifests WG, as well as the Arrikto distribution

**Comment by terrytangyuan** [comment]
LGTM for Training WG.

**Comment by pvaneck** [comment]
LGTM for KServe wg /cc @yuzisun

File: releases/release-1.6/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Project Board
+- Contact
+  - [#release](https://app.slack.com/client/T7QLHSH6U/C9V2WT2KV) on slack
+  - [@release-team](https://github.com/orgs/kubeflow/teams/release-team) on GitHub
+
+### TL;DR
+
+The 1.6 release cycle is proposed as follows
+
+- **Wednesday, Mar 16th 2022**: Week 1 - Release Cycle Begins
```
Comment (thesuperzapper) [inline releases/release-1.6/README.md:19]: I assume you mean May, not march

Comment (annajung) [reply releases/release-1.6/README.md:19]: March is correct here. The proposal is that the 1.6 release process started a week after the 1.5 was released. All other deadlines here are based on the fact that the release cycle began on March 16th.

**Comment by annajung** [comment]
Based on feedback from today's community meeting regarding the docs deadline, the docs deadline has been moved to week 18 to allow distribution owners to have time to adjust docs after testing.

Thank you all for the feedback

/hold cancel

**Comment by surajkota** [comment]
lgtm from AWS distribution side

**Comment by zijianjoy** [comment]
Sounds good to me for Google Cloud, thank you Anna!

**Comment by jbottum** [comment]
@annajung we need to update the list below, can you please double check the dates below?  
                                                              
Wednesday, Mar 16th, 2022: Week 1 - Release Cycle Begins
Wednesday, Jun 15, 2022: Week 12 - Feature Freeze
Wednesday, Jun 29, 2022: Week 14 - Manifests Testing Starts
Wednesday, Jul 6 , 2022: Week 15 - Manifests Testing Ends
Wednesday, Jul 6, 2022: Week 15 - Docs Update Starts
Wednesday, Jul 6, 2022: Week 15 - Distribution Testing Starts
Wednesday, Jul 27, 2022: Week 18 - Distribution Testing Ends
Wednesday, Jul 27,  2022: Week 18 - Docs Update Ends
Wednesday, Aug 3, 2022: Week 19 - Kubeflow v1.6 Released

**Comment by annajung** [comment]
Hi @jbottum there is a PR already open to update the release timeline, see https://github.com/kubeflow/community/pull/561

**Pull request merged**